doc/engineering-reference/src/daylighting-and-window-calculations/window-heat-balance-calculation.tex

```diff
@@ -1039,13 +1044,15 @@ \subsection{Heat Balance Equations for Shading Device and Adjacent Glass}\label{
 The heat balance equation for the shading layer surface facing the zone air is:
 
 \begin{equation}
-{E_i}{\varepsilon_{sh}} - {\varepsilon_{sh}}\sigma \theta_6^4 + {k_{sh}}\left( {{\theta_5} - {\theta_6}} \right) + {h_i}\left( {{T_i} - {\theta_6}} \right) + {S_{sh,2}} = 0
+{E_i}{\varepsilon_{sh}} - {\varepsilon_{sh}}\sigma \theta_6^4 + {k_{sh}}\left( {{\theta_5} - {\theta_6}} \right) + {r_i}{h_i}\left( {{T_i} - {\theta_6}} \right) + {S_{sh,2}} = 0
 \end{equation}
 
 where
 
 \emph{S\(_{sh,2}\)} = solar radiation plus short-wave radiation from lights plus IR radiation from lights and zone equipment absorbed by the zone-side face of the shading layer (W/m\(^{2}\)K).
 
+Note that the coefficient adjustment ratio, \({r_i}\), accounts for highly conductive simple glazing exterior windows. This adjustment ratios remains 1 for other cases.
```

Comment:
I think I am understanding better the implementation here now. It seems that the key would be how the `r_o` and `r_i` are implemented in this feature---it looks the codes does demonstrated some strategies on how to coming up with proper `r_o` and `r_i` values. 

I can see the current addition to the Engineering Reference explained how these two variables are used. However, the Engineering Reference document definitely should have some added records on how these `r_o` and `r_i` values are obtained at different places (at least roughly corresponding to the logics in the code changes)---which I think be the most important information to be added in this documentation.

Reply:
Thanks, @jcyuan2020, I added some notes on how the adjustment ratios are computed. One example is as follows: [code link](https://github.com/yujiex/EnergyPlus/blob/ec983bb16726c4d48907662c79301525ff2b3efa/doc/engineering-reference/src/daylighting-and-window-calculations/window-heat-balance-calculation.tex#L83)